dadjokes.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+
+	"github.com/rs/zerolog/log"
+)
+
+func dadjoke() string {
+	return httpRequest()
+}
+
+func httpRequest() string {
```

Comment:
function name is misleading. recommend renaming to indicate it is scoped specifically to dad jokes.